packages/typedoc-plugin-appium/lib/logger.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Adapted from `@knodes/typedoc-pluginutils`
+ *
+ * Portions Copyright (c) 2022 KnodesCommunity
+ * Licensed MIT
+ *
+ * @module
+ * @see https://github.com/KnodesCommunity/typedoc-plugins/blob/ed5e4e87f5d80abf6352e8de353ea376c4f7db6d/packages/pluginutils/src/plugin-logger.ts
+ *
+ */
+
+import {format} from 'node:util';
+import {Logger, LogLevel} from 'typedoc';
+
+const LogMethods = {
+  [LogLevel.Error]: 'error',
+  [LogLevel.Warn]: 'warn',
+  [LogLevel.Info]: 'info',
+  [LogLevel.Verbose]: 'verbose',
```

Comment:
I would prefer to have debug level as well

Reply:
There's no debug level, since these levels come from typedoc.  typedoc only has "verbose"